forui/lib/src/widgets/bottom_navigation_bar/bottom_navigation_bar_item.dart

```diff
@@ -7,26 +7,43 @@ import 'package:forui/forui.dart';
 
 /// A [FBottomNavigationBar] item.
 class FBottomNavigationBarItem extends StatelessWidget {
+  static ValueWidgetBuilder<FBottomNavigationBarData> _icon(SvgAsset icon) => (_, data, __) => icon(
+        height: data.itemStyle.iconSize,
+        colorFilter: ColorFilter.mode(
+          data.selected ? data.itemStyle.activeIconColor : data.itemStyle.inactiveIconColor,
+          BlendMode.srcIn,
+        ),
+      );
+
   /// The style.
   final FBottomNavigationBarItemStyle? style;
 
-  /// The icon.
-  final SvgAsset icon;
+  /// The icon's builder.
+  final ValueWidgetBuilder<FBottomNavigationBarData> iconBuilder;
 
   /// The label.
   final String label;
 
   /// Creates a [FBottomNavigationBarItem].
-  const FBottomNavigationBarItem({
-    required this.icon,
+  FBottomNavigationBarItem({
+    required this.label,
+    required SvgAsset icon,
+    this.style,
+    super.key,
+  }) : iconBuilder = _icon(icon);
+
+  /// Creates a [FBottomNavigationBarItem] with a custom icon.
+  const FBottomNavigationBarItem.customIcon({
```

Comment:
If we convert a label to a widget, I think we can just name this `FBottomNavigationBarItem.custom`.